coordinator/internal/types/get_task.go

```diff
@@ -4,13 +4,16 @@ package types
 type GetTaskParameter struct {
 	ProverHeight uint64 `form:"prover_height" json:"prover_height"`
 	TaskTypes    []int  `form:"task_types" json:"task_types"`
+	TaskID       string `form:"task_id,omitempty" json:"task_id,omitempty"`
```

Comment:
prover how to know the task_id when it request a task from coordinator?

Reply:
It is a place holder prepared for resolving issue #1670

Reply:
At normal time, it is empty.   Coordinator can assign any task.

Reply:
ok, got it